mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -213,6 +213,22 @@ def process_code_coverage(key, value, timestamp):
 
         p = start_conn_process()
 
+        conn_process_started = False
+        try:
+            (key, value, timestamp) = event_queue.get(timeout=self.options.process_start_timeout)
+
+            if key == '__conn_process_start':
```

Comment:
Future suggestion. In order be robust against https://github.com/ARMmbed/htrun/pull/131/files#diff-9e6e1b40e765e6a2609375222e35117dR132 advisory. This can be put under a loop for ```self.options.process_start_timeout```.